package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "publiclab-editor",
-  "version": "2.2.0",
+  "version": "3.2.0",
```

```suggestion
  "version": "3.0.0",
```

Here we'd zero out the minor and patch numbers if we are doing a major. Is it 100% required though? Does our code break if you have a Bootstrap 3 template? Just checking!

---

Yes @jywarren  this is a breaking change and CSS will break for people on lower bootstrap version.